go/test/endtoend/vtgate/queries/misc/misc_test.go

```diff
@@ -174,8 +174,8 @@ func TestIntervalWithMathFunctions(t *testing.T) {
 	// Set the time zone explicitly to UTC, otherwise the output of FROM_UNIXTIME is going to be dependent
 	// on the time zone of the system.
 	mcmp.Exec("SET time_zone = '+00:00'")
-	mcmp.AssertMatches("select '2020-01-01' + interval month(DATE_SUB(FROM_UNIXTIME(1234), interval 1 month))-1 month", `[[CHAR("2020-12-01")]]`)
-	mcmp.AssertMatches("select DATE_ADD(MIN(FROM_UNIXTIME(1673444922)),interval -DAYOFWEEK(MIN(FROM_UNIXTIME(1673444922)))+1 DAY)", `[[DATETIME("2023-01-08 13:48:42")]]`)
+	mcmp.AssertMatches("select '2020-01-01' + interval month(date_sub(FROM_UNIXTIME(1234), interval 1 month))-1 month", `[[CHAR("2020-12-01")]]`)
```

Comment:
@systay It's because we normalize explicit function names here to lowercase for explicitly known parser implementations and I didn't want to diverge from that here. Similarly here, for example `month` also was emitted as lowercase. 

It's also how MySQL itself normalizes it if you use it for example in a `default` in `create table` if you then issue a `show create table` statement. We try to follow that as the canonical form then.

Reply:
Ah yeah, it would also still be valid. I was pretty aggressive though with the find / replace to lowercase it which is why this one was caught as well :smile:.